server/controller/src/test/java/ai/starwhale/mlops/datastore/parquet/ParquetReadWriteTest.java

```diff
@@ -54,154 +53,431 @@ public void setUp() {
+                        .build(),
+                ColumnSchemaDesc.builder().name("tuple_object")
+                        .type("TUPLE")
+                        .elementType((ColumnSchemaDesc.builder()
+                                .type("OBJECT")
+                                .pythonType("placeholder")
+                                .attributes(List.of(
+                                        ColumnSchemaDesc.builder().name("a").type("INT32").build(),
+                                        ColumnSchemaDesc.builder().name("b").type("STRING").build()
+                                ))
+                                .build()))
+                        .build(),
+                ColumnSchemaDesc.builder().name("tuple_list")
+                        .type("TUPLE")
+                        .elementType((ColumnSchemaDesc.builder()
+                                .type("LIST")
+                                .elementType(ColumnSchemaDesc.builder().type("INT32").build())
+                                .build()))
+                        .build(),
+                ColumnSchemaDesc.builder().name("tuple_tuple")
+                        .type("TUPLE")
+                        .elementType((ColumnSchemaDesc.builder()
+                                .type("TUPLE")
+                                .elementType(ColumnSchemaDesc.builder().type("INT32").build())
+                                .build()))
+                        .build(),
+                ColumnSchemaDesc.builder().name("tuple_map")
+                        .type("TUPLE")
+                        .elementType((ColumnSchemaDesc.builder()
+                                .type("MAP")
+                                .keyType(ColumnSchemaDesc.builder().type("STRING").build())
+                                .valueType(ColumnSchemaDesc.builder().type("INT32").build())
+                                .build()))
+                        .build(),
+                ColumnSchemaDesc.builder().name("map")
                         .type("MAP")
                         .keyType(ColumnSchemaDesc.builder().type("STRING").build())
                         .valueType(ColumnSchemaDesc.builder().type("INT32").build())
                         .build(),
-                ColumnSchemaDesc.builder().name("k")
-                        .type("OBJECT")
-                        .pythonType("placeholder")
-                        .attributes(List.of(ColumnSchemaDesc.builder().name("a").type("INT32").build(),
-                                ColumnSchemaDesc.builder().name("b").type("INT32").build(),
-                                ColumnSchemaDesc.builder().name("aa").type("STRING").build()))
+                ColumnSchemaDesc.builder().name("map_object")
+                        .type("MAP")
+                        .keyType(ColumnSchemaDesc.builder()
+                                .type("OBJECT")
+                                .pythonType("placeholder")
```

Comment:
use another name for 'pythonType' so that we can find some bug which be caused by this?